### PR TITLE
Changes to literature review

### PR DESCRIPTION
**Pull request by dmurvihill**

### CODE REVIEW

**Comment by mheon** [comment]
My master is missing a file (ngram-diagram.png). Can you add a commit to fix this?

**Comment by dmurvihill** [comment]
Done.

**Comment by dmurvihill** [comment]
Please merge.

**Comment by mheon** [comment]
This looks good, aside from my comments on the introduction. If you remove that commit from the branch, I'll gladly merge, and we can look at the intro separately

File: doc/src/litreview.ltx [hunk 1]
```diff
@@ -43,44 +41,45 @@ This section summarizes our review of existing literature in the area of
...
+academic dishonesty.
+
+Students may also host code they wrote for a class on \textit{GitHub}, where
+classmates or future students may copy the code without permission or
+attribution. In such cases, almost anyone familiar with academia would agree
+that the student who copied the given code was guilty of unauthorized copying.
+However, many go further, arguing that the student who originally hosted
+the code is guilty of academic dishonesty for enabling copying to take
+place~--- even if that student is unaware that the copying occurred
+\cite{cosma06}.
```
Comment (mheon) [inline doc/src/litreview.ltx]: I like this, flows a lot more smooth than the original

**Comment by mheon** [comment]
Not pulling ATM because it include #22 which I have some comments on, but otherwise seems fine

**Pull request merged**